pkg/resources/grant_database_role_acceptance_test.go

```diff
@@ -57,6 +57,39 @@ func TestAcc_GrantDatabaseRole_databaseRole(t *testing.T) {
 	})
 }
 
+func TestAcc_GrantDatabaseRole_issue2402(t *testing.T) {
+	databaseName := "DB_TEST_DB_DEV"
```

Comment:
I know those are the values from the issue, but I think they should be randomized. Some of the tests are failing because of the hardcoded values.